nni/retiarii/hub/README.md

```diff
@@ -10,20 +10,78 @@ For further motivations and plans, please see https://github.com/microsoft/nni/i
 
```

Comment:
Wait, this file is source, not doc?

Reply:
Space hub is in a preview state and hidden from documentation. I think we'll announce it shortly after 1 or 2 more PRs are merged.